### PR TITLE
Ops 3221 send rc message on ref prod deployments

### PR DESCRIPTION
**Pull request by aimee-889**
# Description
Send a notification about the status (succeeded/failed) of a rollout to rocketchat into the "Release" channel

## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/OPS-3221

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Review by BlexToGo — DISMISSED** [review]
Test with another channel was successful

File: .github/workflows/host.yml [hunk 1]
```diff
@@ -142,11 +142,11 @@ jobs:
...
       - name: Send Notification to Rocket Chat
         uses: RocketChat/Rocket.Chat.GitHub.Action.Notification@1.1.1
-        if: always()
+        if: always() && (contains(inputs.host_name, 'ref')) || (contains(inputs.host_name, 'prod'))
         with:
           type: ${{ job.status }}
           job_name: 'Rollout of ${{ inputs.host_name }} triggered by ${{ github.actor }}:'
           url: ${{ secrets.RC_WEBHOOK }}
-          channel: '#releases'
+          channel: '#release'
```
Comment (BlexToGo) [inline .github/workflows/host.yml:150]: Necessary renaming of the rocket chat channel is already done
> Room name changed to: release by sophie.krueger

Comment (aimee-889) [reply .github/workflows/host.yml:150]: Here is only the s dropped. Actually the room name could have stayed the same. Only in the github action the channel name had to be in lower case. Nevertheless RC would push the message to the upper case channel

File: .github/workflows/host.yml [hunk 2]
```diff
@@ -142,11 +142,11 @@ jobs:
           rm -rf  /config
       - name: Send Notification to Rocket Chat
         uses: RocketChat/Rocket.Chat.GitHub.Action.Notification@1.1.1
-        if: always()
+        if: always() && (contains(inputs.host_name, 'ref')) || (contains(inputs.host_name, 'prod'))
```
Comment (BlexToGo) [inline .github/workflows/host.yml:145]: Does the `always()` still serve any purpose? If not, I would suggest simplifying this if statement for readability to
```suggestion
        if: contains(inputs.host_name, 'ref') || contains(inputs.host_name, 'prod')
```
And was it possible to quickly test if the `contains()` expression works as expected? For example, by deploying to `infra` and `dev` while having the if statement
```
        if: contains(inputs.host_name, 'infra')
```
and checking afterwards if only a message for the `infra` environment was sent to the test rocket chat channel.

**Review by BlexToGo — CHANGES_REQUESTED** [review]
Open question if the if statement of the `Send Notification to Rocket Chat` can be simplified

**Review by BlexToGo — APPROVED** [review]
Conversation about `always()` is resolved, workflow is tested, approved for merge into main

**Pull request merged**